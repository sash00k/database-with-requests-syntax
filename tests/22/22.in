get records where course is set as teachers
get records from teachers where groups contains 22129