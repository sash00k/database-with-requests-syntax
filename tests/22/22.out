Teacher Власов Г. В. on course Ан.геом. Works with 2 groups and 2 students.
Student Ударников В. П. from group 17121, ММФ. Also teaches Программирование-2 in 1 groups and mentors 0 students.